flixel/system/frontEnds/BitmapFrontEnd.hx

```diff
@@ -123,18 +123,18 @@ class BitmapFrontEnd
 	}
 
 	/**
-	 * Generates a new BitmapData object (a colored rectangle) and caches it.
+	 * Creates a new FlxGraphic object (a colored rectangle) and caches it.
 	 *
-	 * @param	Width	How wide the rectangle should be.
-	 * @param	Height	How high the rectangle should be.
-	 * @param	Color	What color the rectangle should be (0xAARRGGBB)
-	 * @param	Unique	Ensures that the bitmap data uses a new slot in the cache.
-	 * @param	Key		Force the cache to use a specific Key to index the bitmap.
-	 * @return	The BitmapData we just created.
+	 * @param	width	How wide the rectangle should be.
+	 * @param	height	How high the rectangle should be.
+	 * @param	color	What color the rectangle should be (0xAARRGGBB).
+	 * @param	unique	Ensures that the bitmap data uses a new slot in the cache.
+	 * @param	key		Force the cache to use a specific Key to index the bitmap.
+	 * @return	The FlxGraphic we just created.
```

Comment:
here's an example of the formatting, and lets just call it a "graphic" not a "`FlxGraphic` object"
```suggestion
	 * @param   width   How wide the rectangle should be.
	 * @param   height   How high the rectangle should be.
	 * @param   color     What color the rectangle should be (0xAARRGGBB).
	 * @param   unique  Ensures that the bitmap data uses a new slot in the cache.
	 * @param   key       Force the cache to use a specific Key to index the bitmap.
	 * @return  The created graphic.
```

All of these suggestions can and should be committed using the dropdown below each. Please update the other methods